docs/cli/cozy-stack_settings.md

```diff
@@ -0,0 +1,44 @@
+## cozy-stack settings
```

Comment:
@aeris @nledez I think you can be interested by this command to update the `tracking` setting of some instances